packages/terra-list/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed DnD issue for `terra-list` inside `terra-slide-panel`.
```

Comment:
This is too vague. What is the issue? it's also better to spell out "drag and drop" than use an abbreviation.

Reply:
Identified a bug where the position of dragged or selected items is lost when using Terra List with draggable items inside a Terra Slide Panel component.